platform/.run/EducationBenchmarking.Platform.Api.Benchmark.run.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="EducationBenchmarking.Platform.Api.Benchmark" type="AzureFunctionsHost" factoryName="Azure Functions host">
+    <option name="EXE_PATH" value="$PROJECT_DIR$/src/apis/EducationBenchmarking.Platform.Api.Benchmark/bin/Debug/net6.0/EducationBenchmarking.Platform.Api.Benchmark.dll" />
+    <option name="PROGRAM_PARAMETERS" value="host start --pause-on-error --port 7072 --csharp" />
+    <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$/src/apis/EducationBenchmarking.Platform.Api.Benchmark/bin/Debug/net6.0" />
+    <option name="PASS_PARENT_ENVS" value="1" />
+    <option name="USE_EXTERNAL_CONSOLE" value="0" />
+    <option name="USE_MONO" value="0" />
+    <option name="RUNTIME_ARGUMENTS" value="" />
+    <option name="PROJECT_PATH" value="$PROJECT_DIR$/src/apis/EducationBenchmarking.Platform.Api.Benchmark/EducationBenchmarking.Platform.Api.Benchmark.csproj" />
+    <option name="PROJECT_EXE_PATH_TRACKING" value="1" />
+    <option name="PROJECT_ARGUMENTS_TRACKING" value="0" />
+    <option name="PROJECT_WORKING_DIRECTORY_TRACKING" value="1" />
+    <option name="PROJECT_KIND" value="RunnableProjectKind(name=AzureFunctions)" />
+    <option name="PROJECT_TFM" value="net6.0" />
+    <option name="FUNCTION_NAMES" value="" />
+    <browser url="http://localhost:7072" />
+    <method v="2">
+      <option name="BuildFunctionsProject" enabled="true" />
+      <option name="RunAzuriteTask" enabled="true" />
```

Comment:
Might be an open question to the team about how they have Azurite setup? I tend to run Azurite in docker.

Oddly you don't need Azurite running to run func apps locally,

Reply:
Hmm. I didn't even notice that in there. Not quite sure when Rider would have even asked me to configure this - I probably just clicked a 'sure, OK' button!

Reply:
Rider doesn't ask... it just does. It burnt me about 6 months when the Azurite I thought was being used, wasn't the one I thought!!